### ISSUE TITLE
Jest on windows

### ISSUE THREAD

**Issue by Joe-Withey** [janko-m/vim-test#290]
I've found for Jest to work with vim-test on windows I have to use the following config

```
     function! JestTransform(cmd) abort
      let correct_path = substitute(a:cmd, '\\', '\\\\', 'g')
      let correct_call = substitute(correct_path, '\/', '\\', 'g')
      return correct_call
     endfunction

   let g:test#custom_transformations = {'jest': function('JestTransform')}
   let g:test#transformation = 'jest'
```

This is due to windows paths and Jests use of regex.

OS: Windows 8
Editor: gVim 8

**Comment by janko-m** [comment]
Yeah, it was necessary to add Windows support to the .NET runner as well (https://github.com/janko-m/vim-test/pull/277). I would l like to have some generalized helpers for this that all other runners can reuse so that they automatically work well on Windows.

**Comment by jmzagorski** [comment]
I agree with @janko-m since this is also an issue in the javascript karma runner. I am sure it pops up in other ones as well. I will try to help by looking into to the code more when I get some time if nobody has a solution by then.
